### PR TITLE
net-glb: Added support for regional external HTTP(s) load balancing

### PR DESCRIPTION
**Pull request by rosmo**
Added tests too.

### CODE REVIEW

File: modules/net-glb/backend-services.tf [hunk 1]
```diff
@@ -21,10 +21,14 @@ locals {
     for k, v in coalesce(var.backend_services_config, {}) :
     k => v if v.bucket_config != null
   }
-  backend_services_group = {
+  backend_services_group = var.region == null ? {
     for k, v in coalesce(var.backend_services_config, {}) :
     k => v if v.group_config != null
-  }
+  } : {}
+  regional_backend_services_group = var.region != null ? {
```
Comment (ludoo) [inline modules/net-glb/backend-services.tf:28]: nit: can we use a single local, and apply the condition where it's set in the resource?

File: modules/net-glb/backend-services.tf [hunk 2]
```diff
@@ -208,3 +212,150 @@ resource "google_compute_backend_service" "group" {
     }
   }
 }
+
```
Comment (ludoo) [inline modules/net-glb/backend-services.tf:215]: I would move the regional resources to a `regional-backend-services.tf` file

**Review by ludoo — CHANGES_REQUESTED** [review]
Can you add a section with usage example to the README?

File: modules/net-glb/backend-services.tf (hunk 2 above)
Comment (rosmo) [reply modules/net-glb/backend-services.tf:215]: Should I move all regional resources to `regional-` files? My original idea was to keep the resources in same files where it made sense (eg. the file is short enough for comprehension) given the resource config is almost 1:1.

Comment (ludoo) [reply modules/net-glb/backend-services.tf:215]: Hmmm, maybe only for "fat files" like backends then?

**Review by ludoo — APPROVED** [review]

File: modules/net-glb/backend-services.tf (hunk 2 above)
Comment (rosmo) [reply modules/net-glb/backend-services.tf:211]: I moved the backends to regional file.

File: modules/net-glb/backend-services.tf (hunk 1 above)
Comment (rosmo) [reply modules/net-glb/backend-services.tf]: Fixed!

**Comment by rosmo** [comment]
> Can you add a section with usage example to the README?

Added a small section with an example.

**Pull request merged**